pkg/unvanquished_src.dpkdir/translation/game/fr.po

```diff
@@ -3206,7 +3204,7 @@ msgstr "%s%s ^*a été déchiqueté par %s%s%s^*'s %s"
 #: src/cgame/cg_event.cpp:481
 msgctxt "female"
 msgid "%s%s ^*was shredded by %s%s%s^*'s %s"
-msgstr "%s%s ^*a été déchiquetée par %s%s%s^*'s %s"
+msgstr "%s%s ^*a été déchiquetée par le %s de %s%s%s^*"
```

Comment:
This totally break the string because ordering is different.

Reply:
`%1$s`, `%2$s` etc. should be working now by the way, although we haven't done anything to make that accessible.

Reply:
What do you mean with “we haven't done anything to make that accessible”?

Reply:
I mean to make it apparent to translators. Like writing any instructions or rewriting some strings to serve as examples.

Reply:
The `%1$s` syntax doesn't respect ordering (whatever the number, first string is used):

- https://github.com/Unvanquished/Unvanquished/pull/2885